frappe/public/js/frappe/views/reports/query_report.js

```diff
@@ -832,6 +832,7 @@ frappe.views.QueryReport = class QueryReport extends frappe.views.BaseList {
 		if (this.raw_data.add_total_row) {
 			data = data.slice();
 			data.splice(-1, 1);
+			this.$page.find('.layout-main-section').css('--report-total-height', '310px');
```

Comment:
is this a default value we are hardcoding?

Reply:
Default is `275px` but when `add_total_row` is set then I am changing it to `310px` to make total row visible.